pkg/apiserver/registry/networkpolicy/clustergroupmember/rest.go

```diff
@@ -62,12 +62,11 @@ func (r *REST) Get(ctx context.Context, name string, options *metav1.GetOptions)
 			effectiveIPBlocks = append(effectiveIPBlocks, ipb.CIDR)
 		}
 		memberList.EffectiveIPBlocks = effectiveIPBlocks
-	} else {
-		effectiveMembers := make([]controlplane.GroupMember, 0, len(groupMembers))
```

Comment:
Is it still better to keep the code that allocates required size in one shot to be a bit more efficient?

Reply:
Of course. My bad. Fixed